README.md

```diff
@@ -2,6 +2,33 @@
 
 [![Check build](https://github.com/fedora-ci/ciboard/actions/workflows/build.yaml/badge.svg)](https://github.com/fedora-ci/ciboard/actions/workflows/build.yaml)
 
+## Architecture
+
+The following diagram lays out the high-level architecture of CI Dashboard and its component packages:
+
+```mermaid
```

Comment:
I like it. Is there some special tool, that helps to draw these diagrams?

Reply:
Did you manually typed these code? Or it was interactive tooling?

Reply:
I typed this one manually. I don't know if there's a tool, but if I wanted something more complex, I'd use diagrams.net or something like that.